apps/frontend/src/components/cards/controltable/ResponsiveRowLarge.vue

```diff
@@ -31,6 +31,12 @@
         <slot name="tags" />
       </v-card>
     </v-col>
+
+    <v-col class="pa-0" :cols="0.5">
+      <v-card tile flat class="fill-height">
+        <slot name="runTime" />
+      </v-card>
+    </v-col>
```

Comment:
I'm extremely wary of using partial columns. Last time we merged a PR with partial columns we had to revert it since it breaks word wrap. I don't believe it is a tested/supported feature of Vuetify, it just happens to work sometimes until it doesn't. Last time the bug was with the fact that word wrap doesn't work on columns with half width, and the Result Set row was busting out of its column and breaking the rest of the table. What is it possible to do with this PR and full size columns?